packages/oidc/src/__mocks__/issuer.mocks.ts

```diff
@@ -151,9 +151,9 @@ export const mockDpopTokens = (): TokenEndpointRawResponse => {
   };
 };
 
-export const mockClient = (): IClient => {
+export const mockClient = (clientId = "some client"): IClient => {
   return {
-    clientId: "some client",
+    clientId,
```

Comment:
I thought both client_id and client_secret were generated for dynamic client registration? So would this really be client_name instead?

Reply:
That's correct, the client credentials are generated as part of the dynamic registration flow. However, in the case of the refresh grant, the credentials would have been generated previously during the initial authorization code flow, so here these credentials are just passed from storage. This mock default is just an arbitrary value for an identifier that could have been generated, but could also have been statically registered. Basically at this point in the grant that's irrelevant: either you have both an ID and a secret and you use regular OIDC client authentication (the `Authorization <client id>:<client secret>` header, or you just have an IRI-shaped ID and then you do Solid-OIDC auth.

Reply:
Ooooh, right! Perhaps labelling client_id as "previously issued client_id" or something would make it clearer?